Add ToDo to main.rs for `use bevy_game::GamePlugin` (error[E0432]: unresolved import `bevy_game`)

I just hit this in https://github.com/chickenchicken9/help_game/actions/runs/4583696654/jobs/8094705941#step:6:499.

I have a fix in https://github.com/chickenchicken9/bevy_game_template_fix_package (or feel free to just write your own - I'm still slowly learning GitHub.)

Full error:

error[E0432]: unresolved import `bevy_game`
 --> src/main.rs:8:5
  |
8 | use bevy_game::GamePlugin;
  |     ^^^^^^^^^ use of undeclared crate or module `bevy_game`

I assume this happens because I updated the crate name, which had a ToDo (https://github.com/chickenchicken9/help_game/commit/261e698fd415f03a4e5f391001206f9f965e9439).

--- a/src/main.rs
+++ b/src/main.rs
@@ -5,7 +5,7 @@ use bevy::prelude::*;
 use bevy::window::PrimaryWindow;
 use bevy::winit::WinitWindows;
 use bevy::DefaultPlugins;
-use bevy_game::GamePlugin;  // ToDo: Replace bevy_game with your new crate name.
+use bevy_game::GamePlugin; // ToDo: Replace bevy_game with your new crate name.
 use std::io::Cursor;
 use winit::window::Icon;
 

--- a/src/main.rs
+++ b/src/main.rs
@@ -5,7 +5,7 @@ use bevy::prelude::*;
 use bevy::window::PrimaryWindow;
 use bevy::winit::WinitWindows;
 use bevy::DefaultPlugins;
-use bevy_game::GamePlugin;
+use bevy_game::GamePlugin;  // ToDo: Replace bevy_game:: with the your crate name.
 use std::io::Cursor;
 use winit::window::Icon;
 

--- a/src/main.rs
+++ b/src/main.rs
@@ -5,7 +5,7 @@ use bevy::prelude::*;
 use bevy::window::PrimaryWindow;
 use bevy::winit::WinitWindows;
 use bevy::DefaultPlugins;
-use bevy_game::GamePlugin;  // ToDo: Replace bevy_game:: with the your crate name.
+use bevy_game::GamePlugin;  // ToDo: Replace bevy_game with your new crate name.
 use std::io::Cursor;
 use winit::window::Icon;
 

